packages/repository-json-schema/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@loopback/repository-json-schema",
+  "version": "4.0.0-alpha.1",
+  "description": "Converts TS classes into JSON Schemas using TypeScript's reflection API",
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "build": "npm run build:dist",
+    "build:current": "lb-tsc",
+    "build:dist": "lb-tsc es2017",
+    "build:apidocs": "lb-apidocs",
+    "clean": "lb-clean loopback-repository-json-schema*.tgz dist package api-docs",
+    "prepare": "npm run build && npm run build:apidocs",
+    "pretest": "npm run build:current",
+    "test": "mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js' 'DIST/test/integration/**/*.js' 'DIST/test/acceptance/**/*.js'",
```

Comment:
❗️ `DIST` directory does not exist, this command will become a no-op on case-sensitive file systems (typically on Linux). (Both MacOS and Windows are case-insensitive AFAIK).

Please use `dist` instead of `DIST`.